index.js

```diff
@@ -15,7 +15,6 @@
 var createError = require('http-errors')
 var debug = require('debug')('send')
 var deprecate = require('depd')('send')
-var destroy = require('destroy')
```

Comment:
@dougwilson yeah the `fd` leak shouldn't be a problem since we're now opening and closing the `fd` ourselves instead of relying on `fs.createReadStream()` to do it for us. The "destroy" module was essentially just adding an `.on('open', close)` handler to the stream.

Regarding the min Node version being supported, when do you see that happening? Even bumping the min version to 0.10 would be handy since it will respect the `autoClose` option by default.